meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk.bb

```diff
@@ -37,6 +38,7 @@ FILES_${PN} += " \
     /resin-data \
     ${systemd_unitdir} \
     ${sysconfdir} \
+    /usr/lib/resin-supervisor \
     "
 
 RDEPENDS_${PN} = " \
```

Comment:
I guess the package now depends on healthdog? Though I guess balena already depends on it...

Reply:
Very good catch!

Reply:
Fixed.